src/key.rs

```diff
@@ -127,6 +126,11 @@ impl Serialize for PrivateKey {
     }
 
     fn from_bytes(raw: &[u8]) -> Result<Self, Error> {
+        const FR_SIZE: usize = (Fr::NUM_BITS as usize + 8 - 1) / 8;
```

Comment:
It's not clear to me what the `+ 8 - 1` is doing

Reply:
ceil(size/8) without using floats